docs/configuring-openshift.md

```diff
@@ -0,0 +1,147 @@
+# Interacting with your local OpenShift
+
+The following sections describe how Minishift can assist you in interacting with and configuring your local OpenShift instance provisioned by the Minishift VM. For details about managing the Minishift VM refer to the [Using Minishift](./using.md) section.
```

Comment:
"provisioned by" OpenShift is not 'provisioned by' the Minishift VM. It is either provision by Minishift